apps/docs/src/app/platform/component-docs/platform-list/platform-list-examples/platform-list-border-less-example.component.html

```diff
@@ -1,31 +1,69 @@
-<h3>Imperative Approach</h3>
-<h5>Cozy Mode</h5>
+<h3 id="fd-list-imperative-1">Imperative Approach</h3>
+<h5 id="fd-list-header-1">Cozy Mode</h5>
```

Comment:
why are these id-s added? In normal development we don't add id to every single element in  our markup.